src/rma/js/components/destruction-list/create-modal.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react";
+import Modal from 'react-modal';
+
+import { CsrfInput } from "./csrf";
+import { Input, TextInput } from "./inputs";
+import { SelectInput } from "./select";
+import { countObjectKeys } from "../../utils";
+
+
+function CreateModal(props) {
+    const { checkboxes, modalIsOpen, setIsOpen, reviewers, url, csrftoken } = props;
+
+    const closeModal = () => setIsOpen(false);
+    const selectedUrls = Object.keys(checkboxes).filter(k=>checkboxes[k]);
+    const selectedCount = countObjectKeys(checkboxes);
```

Comment:
No need for a helper here I think:

```suggestion
    const selectedCount = selectedUrls.length;
```